components/gcp/dataproc/create_cluster/component.yaml

```diff
@@ -0,0 +1,76 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: dataproc_create_cluster
+description: |
+  Creates a DataProc cluster under a project.
+inputs:
+  - name: project_id
+    description: >-
+      Required. The ID of the Google Cloud Platform project that the cluster belongs to.
+  - name: region
+    description: 'Required. The Cloud Dataproc region in which to handle the request.'
+  - name: name
+    description: >-
+      Optional. The cluster name. Cluster names within a project must be unique. Names of 
+      deleted clusters can be reused
+    default: ''
+  - name: name_prefix
+    description: 'The regional endpoint to which to direct the request.'
+    default: ''
+  - name: job_name_prefix
```

Comment:
this input is not used anywhere, isn't it?

Reply:
Nice catch. I will remove it.